src/markup/commands/emptyRow.ts

```diff
@@ -0,0 +1,22 @@
+import {EditorSelection, StateCommand} from '@codemirror/state';
+
+const str = '&nbsp;\n\n';
+
+export const insertEmptyRow: StateCommand = ({state, dispatch}) => {
+    const trSpec = state.changeByRange((range) => {
+        const lineFrom = state.doc.lineAt(range.from);
+
+        return {
+            changes: [{from: lineFrom.from, insert: str}],
+            range: EditorSelection.range(
+                range.anchor + str.length,
+                range.head + str.length,
+                range.goalColumn,
+                range.bidiLevel ?? undefined,
+            ),
+        };
+    });
+
+    dispatch(state.update(trSpec));
+    return true;
+};
```

Comment:
I think you can use `replaceOrInsertAfter` helper, like in `insertHRule` command